Network/netx-getting-started-guide.md

```diff
@@ -0,0 +1,273 @@
+{{{
```

Comment:
@chrislittle old post deleted in the PR...not sure what you're asking about the locations and endpoints KB?